ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -201,6 +201,10 @@ public default StorageHandlerInfo getStorageHandlerInfo(Table table) throws Meta
   {
     return null;
   }
+  
+  default String getTableType(org.apache.hadoop.hive.ql.metadata.Table table) {
+    return null;
```

Comment:
should we return `DEFAULT_TABLE_TYPE ` here?

Reply:
I agree, it seems better. Done.